### PR TITLE
added modal

### PR DESCRIPTION
**Pull request by shaksham08**
Added modal 
image

### CODE REVIEW

**Review by sushil-mod — APPROVED** [review]
code, looks fine for me

File: components/modal/style.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  border: none;
+  font-size: inherit;
+  cursor: pointer;
+}
+
+.modal__header {
+  font-size: 2rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```
Comment (t007rushi) [inline components/modal/style.css:31]: can add utility class for this flex-row/flex-col, just suggestion

**Review by t007rushi — APPROVED** [review]
Else modal is quit clean

File: components/modal/style.css (hunk 1 above)
Comment (shaksham08) [reply components/modal/style.css:31]: Sure

**Pull request merged**